src/Microsoft.AspNet.OData.Shared/Common/CommonWebApiResources.Designer.cs

```diff
@@ -1,4 +1,11 @@
-﻿//------------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+// <copyright file="CommonWebApiResources.Designer.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+//------------------------------------------------------------------------------
```

Comment:
Does this auto-generated file need to have the copyright header?

Reply:
Need to exclude the auto-generated files out.